### ISSUE TITLE
webpack: Failed to compile.

### ISSUE THREAD

**Issue by johnmay1** [aspnetboilerplate/module-zero-core-template#207]
Hello,

Downloaded the latest asp.net core and angular template (3.5.0). .Net core build and compiled fine but getting following error during angular compilation,

``C:\Sources\XX` XX\CompanyName.Tools\angular>npm start

> Tools@0.0.0 start C:\Sources\MH USA\MHUSA.Tools\angular
> ng serve --host 0.0.0.0 --port 4200

```
** NG Live Development Server is listening on 0.0.0.0:4200, open your browser on http://localhost:4200/ **
Date: 2018-03-19T18:57:11.611Z
Hash: 8300f5981f9ceb34738b
Time: 17605ms
chunk {inline} inline.bundle.js (inline) 3.85 kB [entry] [rendered]
chunk {main} main.bundle.js (main) 2.91 kB [initial] [rendered]
chunk {polyfills} polyfills.bundle.js (polyfills) 577 bytes [initial] [rendered]
chunk {scripts} scripts.bundle.js (scripts) 1.26 MB [initial] [rendered]
chunk {styles} styles.bundle.js (styles) 1.51 MB [initial] [rendered]
chunk {vendor} vendor.bundle.js (vendor) 852 kB [initial] [rendered]

ERROR in src/shared/helpers/SignalRAspNetCoreHelper.ts(9,9): error TS2322: Type '{ autoConnect: true; connect: undefined; hubs: undefined; qs: string; url: string; }' is not assignable to type 'typeof signalr'.
  Property 'remoteServiceBaseUrl' is missing in type '{ autoConnect: true; connect: undefined; hubs: undefined; qs: string; url: string; }'.


webpack: Failed to compile.`
```

Is anyone else having a similar issue?

Regards,
Hemal

**Comment by ismcagdas** [comment]
@johnmay1 have you used npm to install packages ? If so, can you remove node_moudules folder and ty to install packages iwth yarn ? I assume you have installed the latest version of https://www.npmjs.com/package/abp-web-resources (v3.5).

**Comment by johnmay1** [comment]
Yes, i have updated abp-web-resources to3.5 from 3.3 in package.json file. Removed node_modules folder and try to install using "yarn install" but got the same error.

- John

**Comment by acjh** [comment]
Make [this change](https://github.com/aspnetboilerplate/aspnetboilerplate/pull/3115), which will be fixed in v3.6 with #198:

> Configuring `qs` and `remoteServiceBaseUrl` in SignalRAspNetCoreHelper.ts:
> (can still be improved)
> 
> ```js
> abp.signalr = {
>     autoConnect: true,  // No change
>     connect: undefined, // No change
>     hubs: undefined,    // No change
>     qs: AppConsts.authorization.encrptedAuthTokenName + "=" + ... // No change
>     remoteServiceBaseUrl: AppConsts.remoteServiceBaseUrl,         // Add this
>     startConnection: undefined,                                   // Add this
>     url: '/signalr' // Changed from: AppConsts.remoteServiceBaseUrl + '/signalr'
> };
> ```

**Comment by johnmay1** [comment]
yes, it's fixed the issue. Thanks @acjh.
